.idea/statesNames.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
```

Comment:
Remember, when you've accomplished your task, you should remove any `TODO`s from the code.